### PR TITLE
Deleted Symfony extension

### LINKED ISSUE #41: Move Symfony2 extensions into datagrid-bundle

### PR DESCRIPTION
**Pull request by szymach**
Resolves #41

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open

File: CHANGELOG-2.0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changelog 1.x to 2.0
+
+This is a list changes done in the 2.0 vesion.
+
+## Deleted Symfony extension
+
+Since it was moved to [datagrid-bundle](https://github.com/fsi-open/datagrid-bundle),
+it has been removed from this bundle.
```
Comment (chives) [inline CHANGELOG-2.0.md]: `this bundle` => `the component`

File: README.md [hunk 2]
```diff
@@ -238,28 +237,27 @@ echo $cell->getValue();
 ## Installation ##
 
 This section will describe how to add datagrid into your project.
-First thing is to get DataGrid sources, we recommend you to use [composer](https://github.com/composer/composer#composer---package-management-for-php)
+First thing is to get DataGrid sources, we recommend you to use [composer](https://github.com/composer/composer)
```
Comment (chives) [inline README.md]: `https://github.com/composer/composer` => `https://getcomposer.org/download/`

**Comment by chives** [comment]
We should also remove all useless symfony components from composer.json

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

File: composer.json [hunk 3]
```diff
@@ -11,26 +11,18 @@
...
-        "symfony/translation": "^2.4|^3.0",
-        "symfony/doctrine-bridge": "^2.2|^3.0",
-        "symfony/security-csrf" : "^2.4|^3.0",
-        "symfony/routing" : "^2.2|^3.0",
-        "symfony/http-foundation": "^2.2|^3.0",
-        "symfony/dependency-injection" : "^2.2|^3.0",
-        "phpunit/phpunit": "3.7.*"
+        "doctrine/orm": "^2.5",
+        "doctrine/common": "^2.5",
+        "gedmo/doctrine-extensions": "^2.3.10|^2.4",
```
Comment (chives) [inline composer.json]: `"^2.3.10"` is enough

Comment (szymach) [reply composer.json]: But that does not allow for `2.4` we should check all newest versions, no?

Comment (chives) [reply composer.json]: https://getcomposer.org/doc/articles/versions.md#caret-version-range-

**Pull request merged**